adet/data/dataset_mapper.py

```diff
@@ -34,7 +34,8 @@ class DatasetMapperWithBasis(DatasetMapper):
 
     def __init__(self, cfg, is_train=True):
         super().__init__(cfg, is_train)
-
+        
+        self.crop = self.augmentation[0]
```

Comment:
I think this is not correct when `if cfg.INPUT.CROP.ENABLED and is_train` is `False`.

Reply:
Hi, would you also help review the change above.  Thank you.